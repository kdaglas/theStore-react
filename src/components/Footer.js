import React from "react";

const Footer = props => {
  return (
    <footer>
      <p>
        Â©2018 <a href="#">theStore</a>
        <br />
        Designer - Douglas Kato.
        <br />
        Project by <a href="#">Andela</a>.
      </p>
      <p />
      <p>
        <small>
          <a href="#">Store Manager</a> is a web application that helps store
          owners <br />
          manage sales and product inventory records.
        </small>
      </p>
    </footer>
  );
};

export default Footer;
