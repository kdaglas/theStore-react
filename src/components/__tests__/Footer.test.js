import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';


describe('Footer Component', () => {
  it('should render Footer without crushing', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
