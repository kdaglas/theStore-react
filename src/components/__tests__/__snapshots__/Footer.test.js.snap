// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render Footer without crushing 1`] = `ShallowWrapper {}`;

exports[`Shared Components should render Footer without crushing 1`] = `ShallowWrapper {}`;
